omod-2.0/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs2_0/ConceptAttributeTypeController2_0Test.java

```diff
@@ -85,6 +85,16 @@ public void shouldListAllConceptAttributeTypes() throws Exception {
 		Assert.assertEquals(getAllCount(), Util.getResultsSize(result));
 	}
 
+	@Test
+   	public void shouldPurgeConceptAttributeType() throws Exception {
+    	final String UUID = "9516cc50-6f9f-11e0-8414-001e378eb67f";
+       	Assert.assertNotNull(service.getConceptAttributeTypeByUuid(UUID));
```

Comment:
worth looking into the indention

Reply:
@kdaud , I am going to fix my indentation right here on gitHub since it is still stubborn.

Reply:
@kdaud I have fixed the indentation for the purge method.

Reply:
@kdaud I have realized that I need to indent the method for returning list of concept attribute types. The PR was already merged. Should I just go ahead and made another PR for indentation? I need your thoughts about this.